### PR TITLE
Choose your color game

### PR DESCRIPTION
**Pull request by Mani-kanta-0539**
# Your checklist for this pull request
yes
## About your game
You have to enter into the color tunnel which you have like.
What is your game about? 
It just funny choosing a color game.You have enter into color tunnel.
How to play your game?
Press WASD to move, J to restart and K to toggle trails. Enter into tunnel with color and alphabets.

## Code
Check off the items that are true.
- [x] The game was made using the [Sprig editor](https://editor.sprig.hackclub.com/).
- [x] The game is placed in the in the [`/games` directory](https://github.com/hackclub/sprig/tree/main/games).
- [x] The code is significantly different from all other games in the [Sprig gallery](https://sprig.hackclub.com/gallery) (except for games labeled "demo").
- [x] The game runs without errors. 
- [x] The name of the file/game contains only alphanumeric characters, `-`s, or `_`s.
- [x] The game name is not the same as the others from [gallery](https://sprig.hackclub.com/gallery)

## Image (If an image is used)

- [x] The image is in the [`/games/img` directory](https://github.com/hackclub/sprig/tree/main/games/img).
- [x] The name of the image matches the name of your file. Example: `sokoban_plus.js` -> `sokoban_plus.png`.

> Thanks for PR!

### CODE REVIEW

**Comment by LucasHT22** [comment]
Thanks for PR! which of the two files is the game?

File: games/UNTITLED (1).js [hunk 1]
```diff
@@ -0,0 +1,187 @@
+const player = "p";
```
Comment (Arpan-206) [inline games/UNTITLED (1).js:1]: Author and title comment is missing.

Comment (Arpan-206) [inline games/UNTITLED (1).js:1]: Author and title comment is missing.

**Review by Arpan-206 — CHANGES_REQUESTED** [review]

**Comment by Arpan-206** [comment]
In the file `games/UNTITLED (1).js` maybe try adding a score or timer?

**Comment by grymmy** [comment]
@Mani-kanta-0539 I am closing this PR.  **You are free to reopen this PR yourself** (or reach out to me directly!) and continue advocating for this code landing, but it seems that it hasn't been acted upon for 3 months.

**Pull request closed without merging**